### PR TITLE
Create namespace/sa/rolebinding

### LINKED ISSUE #119: [RFE] Create namespaces
**Is your feature request related to a problem? Please describe.**

k8s-netperf should take care of creating/removing the required namespaces so we don't need to create it externally.

### PR DESCRIPTION
**Pull request by jtaleric**
## Type of change

- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Caving and finally adding the code to create the namespace, service account and role binding to allow for `hostNetwork`.

## Related Tickets & Documents

- Related Issue #119 
- Closes #119 

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please describe the System Under Test.
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: pkg/k8s/kubernetes.go [hunk 1]
```diff
@@ -67,6 +69,56 @@ const clientAcrossRole = "client-across"
...
+		}
+	}
+	rBinding := &v1.RoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      sa,
+			Namespace: namespace,
+		},
+		RoleRef: v1.RoleRef{
+			Kind: "ClusterRole",
+			Name: "system:openshift:scc:hostnetwork",
```
Comment (rsevilla87) [inline pkg/k8s/kubernetes.go:100]: This ClusterRole is openshift specific, do we want to create a new one to make k8s-netperf compatible with other k8s distros?

Comment (jtaleric) [reply pkg/k8s/kubernetes.go:100]: Good question, I don't see many external contributors using other platforms. Currently I suggest we ship it as is, unless you feel strongly we need to account for this? I already capture if the env is OCP, so I could check for that.

Comment (jtaleric) [reply pkg/k8s/kubernetes.go:100]: Firing up a Kind cluster to kick the tires with non OCP cluster.

Comment (jtaleric) [reply pkg/k8s/kubernetes.go:100]: Actually this worked ootb with kind... no issues.

**Pull request merged**